packet/routes/api.py

```diff
@@ -6,8 +6,8 @@
 @app.route("/api/v1/<member_username>/sign/<packet_username>")
 @auth.oidc_auth
 @before_request
-def sign(member_username, packet_username, uid):
-    if uid != member_username:
+def sign(member_username, packet_username, info):
+    if info.uid != member_username and "eboard-evaluations" not in info.member_info.group_list:
```

Comment:
This will raise an `AttributeError` when run under the freshman realm (or if `member_info` is not otherwise present).

Reply:
now that python 3.8 is in development, we have discussion on [PEP 505](https://www.python.org/dev/peps/pep-0505/#pep-deferral) again. If it makes it in, would be great to be able to do `not in info?.member_info?.group_list:`

Reply:
That does look pretty neat but I wouldn't block this PR on the PEP landing/3.8 being released :smile: